plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -56,8 +57,10 @@ public static QueryRunner createSqlServerQueryRunner(
                 .setExtraProperties(extraProperties)
                 .build();
         try {
-            queryRunner.installPlugin(new TpchPlugin());
-            queryRunner.createCatalog("tpch", "tpch");
+            if (Iterables.size(tables) != 0) {
```

Comment:
Seems redundant and we typically not do this. Let's drop it for simplicity

Reply:
Done!